benches/setup.rs

```diff
@@ -50,29 +50,44 @@ impl Mergeable for BenchTestContent {
 // CONSTANTS
 // ================================================================================================
 
-// TREE_HEIGHT = 4 and 8 are hard coded in benchmarks
 pub const TREE_HEIGHTS: [u8; 3] = [16, 32, 64];
-pub const NUM_USERS: [usize; 23] = [
+pub const NUM_USERS: [usize; 39] = [
     10_000,
     20_000,
+    30_000,
     40_000,
+    50_000,
     60_000,
+    70_000,
     80_000,
+    90_000,
     100_000,
     200_000,
+    300_000,
     400_000,
+    500_000,
     600_000,
+    700_000,
     800_000,
+    900_000,
     1_000_000,
     2_000_000,
+    3_000_000,
     4_000_000,
+    5_000_000,
     6_000_000,
+    7_000_000,
     8_000_000,
+    9_000_000,
     10_000_000,
     20_000_000,
+    30_000_000,
     40_000_000,
+    50_000_000,
     60_000_000,
+    70_000_000,
     80_000_000,
+    90_000_000,
```

Comment:
The 10M range must be `[n * 10_000_000 for n in range(1,10,2)]`

Reply:
Since the 10M range will take quite long to compute I thought it would better to space them out further. But I'll leave it up to you whether to space more or keep spacing as 10M

Reply:
Cool. I made it [ 10M, 30M, 50M, 70M, 90M ]